tools/mo/unit_tests/mo/bom_test.py

```diff
@@ -68,36 +70,32 @@ def test_bom_file(self):
                 print(f.replace('\\', '/'))
         self.assertTrue(not len(missing_files), '{} files missed in BOM'.format(len(missing_files)))
 
-    @unittest.skip("83076 - test infra change affects bom tests")
     def test_bom_does_not_contain_unittest_files(self):
         for file_name in self.existing_files:
             self.assertFalse(file_name.endswith('_test.py'), 'BOM file contains test file {}'.format(file_name))
 
-    @unittest.skip("83076 - test infra change affects bom tests")
+    # @unittest.skip("83076 - test infra change affects bom tests")
```

Comment:
Remove?

Reply:
thanks for noticing. will remove that

Reply:
same as previous. i think it's `nit` and can be fixed with bunch of other changes if they will be needed.

Reply:
done